.github/workflows/cicd-pipeline.yml

```diff
@@ -9,7 +9,9 @@ concurrency:
 env:
   DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
   ERICA_HOST: ${{ secrets.ERICA_HOST_STAGING }}
-  IMAGE: "steuerlotse_erica:${{ github.run_number }}"
+  IMAGE: "steuerlotse_erica"
+  IMAGE_TAG_LATEST: "steuerlotse_erica:latest"
+  IMAGE_TAG: "steuerlotse_erica:${{ github.run_number }}"
```

Comment:
I would rename those env variables:
- IMAGE_NAME (previous IMAGE)
- IMAGE_TAG_LATEST (this is fine)
- IMAGE_TAG_GITHUB_RUN (previous IMAGE_TAG)